classes/output/core_renderer.php

```diff
@@ -39,14 +39,63 @@ class core_renderer extends \theme_boost\output\core_renderer {
      * Returns the moodle_url for the favicon.
      *
      * This renderer function is copied and modified from /lib/outputrenderers.php
-     * It uses the same logic as Moodle 4.1dev already which introduced a Moodle core favicon setting,
+     *
+     * It checks if the favicon is overridden in a flavour and, if yes, it serves this favicon.
+     * If there isn't a favicon in any flavour set, it continues with the logic from Moodle core.
+
+     * Doing this, it uses the same logic as Moodle 4.1dev already which introduced a Moodle core favicon setting,
```

Comment:
It's not 4.1dev any more, …

Reply:
I corrected this and pushed an updated patch.